test/mixin.js

```diff
@@ -36,4 +36,10 @@ describe('mixin', function() {
         var b = {y: 3, z: 4};
         assert.deepEqual(curried(b), {w: 1, x: 2, y: 3, z: 4});
     });
+
+    it('is curried', function() {
+        var curried = R.mixin(R.__, {w: 1, x: 2});
+        var b = {x: 3, y: 4};
+        assert.deepEqual(curried(b), {w: 1, x: 2, y: 4});
```

Comment:
The name `b` implies that `b` is the second argument to `R.mixin` (the first argument being `a`), yet `b` is actually the first argument. I suggest not naming the argument:

```javascript
assert.deepEqual(curried({x: 3, y: 4}), {w: 1, x: 2, y: 4});
```